Fix invalid Turtle examples

dbs opened the pull request:
Each prefix directive in Turtle requires an ending period.

If the subject is the same, but the predicate and subject are different, then
you have to use a semicolon for a short form statement.

Signed-off-by: Dan Scott <dscott@laurentian.ca>

Commit:
Fix invalid Turtle examples

Each prrefix directives in Turtle requires an ending period.

If the subject is the same, but the predicate and subject are different, then
you have to use a semicolon for a short form statement.

Signed-off-by: Dan Scott <dscott@laurentian.ca>

arojascastro commented:
Thank you! We will review this suggestion in the following days. 

For editors: if the change is accepted, apply in Spanish translation as well.

arojascastro committed:
replace broken url with internet archive url

arojascastro committed:
replace link with internet archive url

adamcrymble committed:
Merge pull request #1218 from programminghistorian/issue-1207

Fix issue 1207

spapastamkou committed:
update about (EN) and apropos (FR) pages as per #951 (#1204)

* update about (EN) and apropos (FR) pages as per #951

* translate 951

* Update about.md

* Update about.md

vgayolrs committed:
new translation Data-wrangling (#1213)

* new translation Data-wrangling

Add new translation of Data wrangling and management in R, text and
images.

* fix internal links

* add review-ticket in YAML

* arrange lesson slug

* delet

* Delete california_nueva_york_poblacion.png

* Delete mississipi_virginia_poblacion.png

* Delete universidades_seculares_antes_1812.png

* Delete universidades_seculares_despues_1812.png

* fix link

https://ggplot2.tidyverse.org/

* fix original lesson slug

It has "_" instead "-"

* move images for spanish translation of data wrangling in R to correct folder

* fix images files endings

* fromat footnotes

* change format text referencing file

* change format text refering file

* Uploading data for administracion de datos

mdlincoln committed:
Merge branch 'linked_data_valid_turtle' of git://github.com/dbs/jekyll into dbs-linked_data_valid_turtle

mdlincoln committed:
correct turtle examples on es LOD introduction lesson
